aggsender/prover/proof_generation_tool.go

```diff
@@ -104,7 +104,7 @@ func (a *AggchainProofGenerationTool) GetRPCServices() []rpc.Service {
 // GenerateAggchainProof generates an Aggchain proof
 func (a *AggchainProofGenerationTool) GenerateAggchainProof(
```

Comment:
Should we rename it to `GenerateSP1StarkProof` perhaps (also update the log messages)?

Reply:
I want to be consistent with the prover API, a their function is called `GenerateAggchainProof`.